plugin/tests/binops.rs

```diff
@@ -89,7 +93,6 @@ fn test_binop_ne() {
     assert!(checker.has_multiple(noneq_msgs, "26:9: 26:15"));
 }
 
-#[test]
```

Comment:
I fail to see why the is not a test anymore. Is this intentional?

Reply:
Sorry, I messed up the merge.